static/templates/user-groups-admin.handlebars

```diff
@@ -1,5 +1,8 @@
 <div id="user-groups-admin" class="settings-section" data-name="user-groups-admin">
     <div class="tip">Anyone in this organization can add user groups.</div>
+    {{#unless is_admin}}
+    <div class="tip">Only user group members and organization administrators can modify the group.</div>
+    {{/unless}}
```

Comment:
Should this not show if the user is a member?

Reply:
Here, this warning needs to be shown for all users who are not admins, since it might be that user is member of one group and not of other, and its beneficial to warn them of the rules.

Reply:
If we show this, minor edits:
"Only group members and organization administrators can modify a group."

Though I think once we sort them into "Your Groups" and "Other groups", we won't need this warning at all.

Reply:
That's right, for the time being, let's display the above warning, when we will sort them, we'll remove that then.